include/render/gles2.h

```diff
@@ -74,6 +74,8 @@ struct wlr_gles2_renderer {
 
 	struct wlr_gles2_buffer *current_buffer;
 	uint32_t viewport_width, viewport_height;
+
+	struct wlr_drm_format_set data_ptr_texture_formats;
```

Comment:
Should this be stored if it's used only when calling `gles2_get_texture_formats`?

Reply:
If it needs to be stored, it's missing a call to `wlr_drm_format_set_finish`

Reply:
Yeah, we need to store it, otherwise we can't return it to the caller and let the caller use it after the function call has returned in `wlr_renderer_get_texture_formats`.

Good catch! Added the `wlr_drm_format_set_finish` call.